src/AppBundle/Resources/views/themes/common/parts/main_tag.html.twig

```diff
@@ -0,0 +1,10 @@
+{% if not content.fields.sponsored_content_disclosure.empty %}
+    {% if shortened|default(false) %}
```

Comment:
I feel like this check/parameter name is not really clear enough. Can't we go without it? For example, if `sponsored_content_disclosure` field value exists and not empty, use it, and then fallback to `'ngsite.layout.sponsored'|trans` ?

Reply:
@emodric the problem here is not if field exists or is it empty or not, but a different output display in full view and in teaser templates.

The logic that sponsored_content_disclosure field is required to display the article as sponsored, both on full and teaser views, but actual text output is different, where on full view we render the text of the sponsored_content_disclosure field and on teasers we display a translated string with is usually much shorter than a long string added via editor.

Unfortunately it is a bit clumsy, but necessary.

Reply:
Then separate it in two templates, rather than having a non-descriptive parameter.

Reply:
Or maybe better, rename the parameter to something else, e.g. use_field.

Reply:
And definitely inverse logic, since shortened: true appears much more than shortened: false

Reply:
Agreed, @mlukac is working on this now.

Reply:
Changes made.